Libraries/LibGUI/Application.h

```diff
@@ -53,8 +54,9 @@ class Application : public Core::Object {
     void register_global_shortcut_action(Badge<Action>, Action&);
     void unregister_global_shortcut_action(Badge<Action>, Action&);
 
-    void show_tooltip(const StringView&, const Gfx::IntPoint& screen_location);
+    void show_tooltip(const StringView&, const Gfx::IntPoint& screen_location, const RefPtr<Widget> tooltip_source_widget);
```

Comment:
Oops, forgot to complain about this one. No need to take a `RefPtr<Widget>` here as that just incurs unnecessary refcount incrementing+decrementing. Just take a `Widget*` :)